rqd/rqd/cuerqd.py

```diff
@@ -135,137 +108,134 @@ def rebootNow(self):
-            runFrame = rqd.compiled_proto.rqd_pb2.RunFrame()
-            runFrame.resource_id = "8888888877777755555"
-            runFrame.job_id = "SD6F3S72DJ26236KFS"
-            runFrame.job_name = "swtest-home-jwelborn_rqd_test"
-            runFrame.frame_id = "FD1S3I154O646UGSNN" + str(random.randint(0, 99999))
-            runFrame.frame_name = "0001-preprocess"
-            # Script output is not buffered due to python -u option
-            runFrame.command = "/net/people/jwelborn/test_python_u -t 5 -e 0"
-            runFrame.user_name = "jwelborn"
-            runFrame.log_dir = "/mcp" # This would be on the shottree
-            runFrame.show = "swtest"
-            runFrame.shot = "home"
-            runFrame.uid = 10164
-            runFrame.num_cores = 50
-
-            rqdHost.launchFrame(runFrame)
-
-        if o == "--test_script_frame_mac":
-            print("Launching script test frame (logs to /tmp)")
-            runFrame = rqd.compiled_proto.rqd_pb2.RunFrame()
-            runFrame.resource_id = "2222222277777755555"
-            runFrame.job_id = "SD6F3S72DJ26236KFS"
-            runFrame.job_name = "swtest-home-jwelborn_rqd_test"
-            runFrame.frame_id = "FD1S3I154O646UGSNN" + str(random.randint(0, 99999))
-            runFrame.frame_name = "0001-preprocess"
-            # Script output is not buffered due to python -u option
-            runFrame.command = "/net/people/jwelborn/test_python_u_mac -t 5 -e 0"
-            runFrame.user_name = "jwelborn"
-            runFrame.log_dir = "/tmp" # This would be on the shottree
-            runFrame.show = "swtest"
-            runFrame.shot = "home"
-            runFrame.uid = 10164
-            runFrame.num_cores = 1
-
-            rqdHost.launchFrame(runFrame)
-
-
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('host', nargs='?', default='localhost', help='RQD hostname (defaults to localhost)')
+    parser.add_argument('-s', action='store_true', help='Print RQD status')
+    parser.add_argument('-v', action='store_true', help='Print RQD version')
+    parser.add_argument('--lp', metavar='coreID', nargs='+', help='Lock the specified cores')
+    parser.add_argument('--ulp', metavar='coreID', nargs='+', help='Unlock the specified cores')
+    parser.add_argument('--lh', action='store_true', help='Lock all cores for the specified host')
+    parser.add_argument('--ulh', action='store_true', help='Unlock all cores for the specified host')
+    parser.add_argument('--nimbyon', action='store_true', help="Turn on 'Not in my back yard' (NIMBY) to stop processing on the specified host")
+    parser.add_argument('--nimbyoff', action='store_true', help="Turn off 'Not in my back yard' (NIMBY) to start processing on the specified host")
+    parser.add_argument('--exit', action='store_true', help='Lock host, wait until machine is idle and then shutdown RQD (Any unlock command will cancel this request)')
```

Comment:
For this and similar strings, I recommend we update the formatting and grammar as follows:

Lock host, wait until machine is idle, and then shutdown RQD. Any unlock command cancels this request.